skbio/io/format/genbank.py

```diff
@@ -721,47 +736,80 @@ def _parse_loc_str(loc_str, length):
     (e) a remote entry identifier followed by a local location
         descriptor (i.e., a-d). e.g. J00194.1:100..202
 
-    TODO:
-    handle (b), (c), (e) cases correctly
+    Notes
+    -----
+    This does not fully handle (e) case. It will discard the remote
```

Comment:
Do you think this documentation makes more sense to put in the user-facing docstring at the top of this module? Seems like a caveat/note that may be important to a user, and they won't see it here unless they're browsing the source code.